support-frontend/assets/pages/digital-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -36,25 +36,32 @@ import { regularPrice as dpRegularPrice, promotion as digitalPackPromotion } fro
 import type { ProductPrices } from 'helpers/productPrice/productPrices';
 import { PriceLabel } from 'components/priceLabel/priceLabel';
 import { PromotionSummary } from 'components/promotionSummary/promotionSummary';
+import { PayPalRecurringButton } from './../../../pages/new-contributions-landing/components/PayPalRecurringButton';
```

Comment:
We should move this component out to the top level

Reply:
Agreed, I'm going to do that in a subsequent PR